apis/management.cattle.io/v3/globaldns_types.go

```diff
@@ -19,6 +19,7 @@ type GlobalDNS struct {
 
 type GlobalDNSSpec struct {
 	FQDN                string   `json:"fqdn,omitempty" norman:"required"`
+	TTL                 string   `json:"ttl,omitempty"`
```

Comment:
we should probably change it to the actual type of the TTL, something like int. So UI/API can do the validation

Reply:
changed.